m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -2133,13 +2141,40 @@ PROCEDURE BUILDSLTRANSFERLIST
           TRANRECIPACCTTYPE(TRANCOUNT)="share"
          TRANRECIPEXPIREDATE(TRANCOUNT)=RECIPLISTEXPIREDATE(EXTRECFOUND)
 
-         BNODEBUGMSG=FORMAT("TMPNUM:#9",TMPNUM)+"   TRANRECIPTYPECHR(TMPNUM):"+ TRANRECIPTYPECHR(TMPNUM)
+         BNODEBUGMSG=FORMAT("SHARECODE TMPNUM:#9  ",TMPNUM)+"(TMPNUM):"+ TRANRECIPTYPECHR(TMPNUM)
```

Comment:
Hi @tkainz , I suggest changing line 2144 from "BNODEBUGMSG=FORMAT("SHARECODE TMPNUM:#9 ",TMPNUM)+"(TMPNUM):"+ TRANRECIPTYPECHR(TMPNUM)" to
 "BNODEBUGMSG=FORMAT("SHARECODE TMPNUM:#9 ",TMPNUM)+"TRANRECIPTYPECHR(TMPNUM):"+ TRANRECIPTYPECHR(TMPNUM)" as currently TRANRECIPTYPECHR(TMPNUM) is described as (TMPNUM)

Reply:
@jordan-keenan Corrected in next commit